packages/core/template_project/app/build.gradle

```diff
@@ -196,5 +196,9 @@ dependencies {
     <% for(const dep of buildGradle.dependencies) { %>
         implementation '<%= dep %>'
     <% } %>
-    implementation 'com.google.androidbrowserhelper:androidbrowserhelper:2.0.0'    
+    if (twaManifest.alphaDependencies && twaManifest.alphaDependencies.enabled) {
+        implementation 'com.google.androidbrowserhelper:androidbrowserhelper:1.4.0-alpha01' 
+    } else {
+        implementation 'com.google.androidbrowserhelper:androidbrowserhelper:2.0.0' 
+    }    
```

Comment:
Do we need both this and the changes in FeatureManager? Surely we just need the one.

Reply:
Ah, so I will remove the changes from `build.gradle`